src/modules/Pam.rb

```diff
@@ -100,7 +100,7 @@ def Enabled(mod)
     end
 
     # Add options or new PAM module
-    # @param string PAM module or option
+    # @param [String] mod PAM module or option
```

Comment:
NP: I prefer the `mod [String]` variant because it is better readable, esp. if the parameter is a normal word. Like
`enable [Boolean] enable feature foo...`
is better than
`[Boolean] enable enable feature foo...`
which looks like a bad English...